R/create.R

```diff
@@ -46,7 +46,7 @@ create_package <- function(path,
   on.exit(proj_set(old_project), add = TRUE)
 
   use_directory("R")
-  use_description(fields, check_name = check_name)
+  use_description(fields, check_name = FALSE)
```

Comment:
This seems to make the `check_name` argument pointless and undoes https://github.com/r-lib/usethis/issues/528. The rationale for having an escape hatch is that people make packages that will never go to CRAN or use the package structure for a data analysis and might have some external naming convention.

Reply:
Note that we're already checking the name on like 40; isn't this check redundant?

Reply:
Oh, yeah, sorry I am very off here.